pkg/driver/node/credentialprovider/awsprofile/aws_profile.go

```diff
@@ -0,0 +1,162 @@
+// Package awsprofile provides utilities for creating and deleting AWS Profile (i.e., credentials & config files).
```

Comment:
This package now accepts `basepath`, `prefix` and `filePerm` settings as arguments (see `Setting` struct) to accommodate changes in `credentialprovider`. There is no other change than that.